### PR TITLE
zh-cn: Format /learn/css using Prettier

### PR DESCRIPTION
**Pull request by queengooborg**
This PR formats the `/learn/css` folder of the Simplified Chinese locale using Prettier.

### CODE REVIEW

File: files/zh-cn/learn/css/howto/css_faq/index.md [hunk 1]
```diff
@@ -215,7 +252,7 @@ div { color: black; }
 
 ## -moz-\*， -ms-\*， -webkit-\*， -o-\* 以及 -khtml-\* 属性有什么用？
 
-这些被称为 *前缀属性* 的属性是 CSS 标准的扩展。这些是出于测试目的的使用，不至于污染标准命名空间，防止标准扩展时产生兼容性问题。
+这些被称为 _前缀属性_ 的属性是 CSS 标准的扩展。这些是出于测试目的的使用，不至于污染标准命名空间，防止标准扩展时产生兼容性问题。
```
Comment (jasonren0403) [inline files/zh-cn/learn/css/howto/css_faq/index.md:255]: remove extra spaces

```suggestion
这些被称为*前缀属性*的属性是 CSS 标准的扩展。这些是出于测试目的的使用，不至于污染标准命名空间，防止标准扩展时产生兼容性问题。
```

File: .prettierignore [hunk 2]
```diff
@@ -19,6 +19,7 @@ build/
 /files/fr/glossary/grid_axis/index.md
 /files/fr/glossary/grid_lines/index.md
 /files/fr/glossary/main_axis/index.md
+/files/zh-cn/learn/css/howto/css_faq/index.md
```
Comment (jasonren0403) [inline .prettierignore:22]: Seems that this file is also formatted 🤔

**Review by jasonren0403 — COMMENTED** [review]
some suggestions

File: .prettierignore (hunk 2 above)
Comment (queengooborg) [reply .prettierignore:22]: It is, but there's one change that Prettier produces with a link in that file that breaks the link...  Seems to be common with this specific file.  :/

Comment (jasonren0403) [reply .prettierignore:22]: Let me open another PR to deal with this file :(

**Review by jasonren0403 — APPROVED** [review]
Other changes are good 👍 , thank you @queengooborg

**Pull request merged**